eng/Version.Details.xml

```diff
@@ -312,9 +312,9 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>bce0a98620c1c5a110b2bba9912f3d5929069c6b</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="3.6.0-2.20166.2" CoherentParentDependency="Microsoft.AspNetCore.Razor.Language">
+    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="3.6.0-3.20205.2" CoherentParentDependency="Microsoft.AspNetCore.Razor.Language">
```

Comment:
FYI this version going down is expected because the latest aspnetcore-tooling build is at 20205.2